README.md

```diff
@@ -1,12 +1,34 @@
 # Cronos Play Unreal SDK Demo
-This repository contains a sample project that uses the Cronos Play Unreal SDK plugin.
+This repository contains sample projects that uses the Cronos Play Unreal SDK plugin.
 For more information, please see the [Cronos Play Unreal SDK plugin repository](https://github.com/crypto-com/play-unreal-plugin).
 
+## basic flow
+- inherit blueprint DefiWalletCoreActor
+- restore wallet with mnemonics
+- transfer amount
+- most of apis are async, receive result with events
+
+## examples
+### simple
+- basic demo 
+- eth address generation with index
+- get balance, transfer etc
+- erc721 fetch, transfer 
+- erc1155 fetch, transfer
+- wallet-connect signing
+- cronoscan, blockscout query
+
+
+### play2earn
+- basic erc721, erc1155 nft game example
+- need to mint tokens beforehand
+
```

Comment:
If README could be more detailed, it could be better... Looks like a draft, and some casual notes instead of a well-written documentation. But anyway, overall looks good to me.

Reply:
ok, i'll check